eZ/Publish/API/Repository/Tests/URLAliasServiceAuthorizationTest.php

```diff
@@ -31,7 +31,9 @@ public function testCreateUrlAliasThrowsUnauthorizedException()
         $urlAliasService = $repository->getURLAliasService();
         $locationService = $repository->getLocationService();
 
-        $location = $locationService->newLocationCreateStruct($parentLocationId);
+        $location = $locationService->loadLocation(
+            $this->generateId('location', 12)
```

Comment:
Please create actual Location instead of relying on hard-coded ID. Right now it's necessary to rely on `1` and `2` only, other data can be either created quickly or retrieved by text identifiers (like login).